### PR TITLE
Vdb 1674 backfill eth b (beta)

### PR DESCRIPTION
**Pull request by elizabethengelman**
To be merged in after #349

### CODE REVIEW

File: environments/backfillEvents.toml [hunk 1]
```diff
@@ -2,7 +2,86 @@
...
     save     = true
-    transformerNames = []
+    transformerNames = [
+        "auction_file",
+        "deal",
+        "dent",
+        "deny",
+        "flip_file_cat",
+        "flip_kick",
+        "rely",
```
Comment (rmulhol) [inline environments/backfillEvents.toml:12]: looks like maybe missing some transformers?

Comment (elizabethengelman) [reply environments/backfillEvents.toml:12]: 🤦‍♀️  nice catch. just pushed up a fix.

**Review by rmulhol — APPROVED** [review]
approving bc I don't _think_ having some transformers missing from the `transformerNames` should matter as long as they're in the exporter and they're config info is in the toml, though may be good to add `tend`, `tick`, and `yank` to that list before merging

**Comment by paytonrules** [comment]
At the risk of being contrary just to be contrary ... oh wait I like doing that.  

If transformerNames don't need to be in the list - why have them? Maybe get rid of it entirely?

**Comment by elizabethengelman** [comment]
@paytonrules that's a really good point - I think that we need them when we're relying on the vdb `compose` to generate `tranformersExporter.go`. We iterate through those names to know what the keys are for the exporters. But, after playing around with the [toml parser](https://github.com/BurntSushi/toml) we're using, I am starting to think it's possible to get those keys in another way. This is a good reminder that I should create a new story to try to address that in the future.

**Pull request merged**